webpack/assets/javascripts/react_app/components/common/AutoComplete/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import Downshift from 'downshift';
+import classnames from 'classnames';
+import { InputGroup } from 'react-bootstrap';
+import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import './AutoComplete.scss';
+
+class AutoComplete extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      containerWidth: '200px',
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ containerWidth: `${this.inputRef.offsetWidth}px` });
```

Comment:
Yeah good call on this. I wanted to have the dropdown match the width of the input box but I guess including the search button is fine and this will make the code simpler. Thanks!